### ISSUE TITLE
Improve reporting of fixed perks

### ISSUE THREAD

**Issue by dcela** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#382]
Is there any interest in working with the maintainer of http://www.bannerlordperks.com to improve the visualization of which perks are fixed?

I sent the owner a message as well.

I think this will also help spread awareness of this mod.

**Comment by JoeFwd** [comment]
Yes, there should be a clearer way for people to understand what perks are fixed and also how they have been fixed.
On top of that, documentation should also be added to make it easier for developers to add their own patches.
Unfortunately, for now, there isn't enough people to conduct these kind of projects.
